### ISSUE TITLE
Fasttemplate: Allow changing colors

### ISSUE THREAD

**Issue by rbps1** [holoviz/panel#2175]
### Current situation
The fast template is an excellent way of building dashboards without spending too much time for html optimisations. However the default colors can be an issues somethings (corporate environment with specific brand colors).

The underlying mechanisms of the fast template do not allow to overwrite color variables using additional css, instead colors have to be passed when creating the template. Currently, this does only support the header colors.

[https://panel.holoviz.org/reference/templates/FastGridTemplate.html#templates-gallery-fastgridtemplate](https://panel.holoviz.org/reference/templates/FastGridTemplate.html#templates-gallery-fastgridtemplate)

![grafik](https://user-images.githubusercontent.com/82584491/114861779-01a06c00-9dee-11eb-85ae-d69f64a415d4.png)

### Feature request: 
Allow to customize all color variables that the fast design template uses when creating a new instance of the fasttemplate.

```
:root {
  --accent-fill-active: #A01346;
  --accent-fill-hover: #DA1A5F;
  --accent-fill-rest: #C01754;
  --accent-foreground-active: #DF3874;
  --accent-foreground-cut-rest: #FFFFFF;
  --accent-foreground-hover: #E55E8E;
  --accent-foreground-rest: #E1477E;
  --background-color: #181818;
  --container-background: #3B3B3B;
  --neutral-divider-rest: #2E2E2E;
  --neutral-fill-active: #262626;
  --neutral-fill-hover: #333333;
  --neutral-fill-input-active: #181818;
  --neutral-fill-input-hover: #181818;
  --neutral-fill-input-rest: #181818;
  --neutral-fill-rest: #2B2B2B;
  --neutral-fill-stealth-active: #212121;
  --neutral-fill-stealth-hover: #262626;
  --neutral-fill-stealth-rest: #181818;
  --neutral-focus: #717171;
  --neutral-focus-inner-accent: #350617;
  --neutral-foreground-active: #E5E5E5;
  --neutral-foreground-focus: #E5E5E5;
  --neutral-foreground-hover: #E5E5E5;
  --neutral-foreground-rest: #E5E5E5;
  --neutral-layer-floating: #4A4A4A;
  --neutral-outline-active: #424242;
  --neutral-outline-hover: #808080;
  --neutral-outline-rest: #5A5A5A;
}
```

Applies to Grid and List version of the template.

### Discourse discussion
[https://discourse.holoviz.org/t/fasttemplate-change-colors/2135](https://discourse.holoviz.org/t/fasttemplate-change-colors/2135)
@MarcSkovMadsen

**Comment by TheoMathurin** [comment]
Hi,

Speaking about changing the colors in those FastTemplates (which are really great), there is one slight problem which concerns the background color of the area where options are shown in MultiChoice widgets. It is completely transparent which yields readability issues, as it is superimposed to the content above and below the widget.

I have looked into the css and it corresponds to ```background-color``` in ```.bk-root .choices__list--dropdown``` in fast_bokeh.css. I was able to change it—for instance setting it to the actual background color of this theme, *i.e.* #F7F7F7—in place from within the navigator. However, in my case changing the local file does not change anything when I reload my app.

Also, in contrast to the native theme, the item on which the mouse hovers is not highlighted, but that's a more minor issue.
